projects/worldskills-angular-lib/src/lib/auth/app-auth.guard.ts

```diff
@@ -1,17 +1,17 @@
 import { Injectable } from '@angular/core';
-import { CanActivate, Router, ActivatedRouteSnapshot, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { CanActivate, Route, ActivatedRouteSnapshot, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { WorldskillsAngularLibService } from '../worldskills-angular-lib.service';
+import { WsiNgAuthService } from './wsi-ng-auth.service';
 import { Observable } from 'rxjs';
-import { AuthService } from '../services/auth.service';
-import { GenericUtil } from '../util/generic-util';
-import { UserModel } from '../models/user.model';
-import { ModuleConfigService } from '../config/module-config.service';
-import { AuthGuardAccess } from '../interfaces/auth-guard-access';
+import { AuthGuardAccess } from './models/auth-guard-access';
+import { User } from './models/user';
+import { GenericUtil } from '../common/util/generic.util';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AppAuthGuard implements CanActivate {
-  constructor(protected authService: AuthService, protected config: ModuleConfigService, protected router: Router) { }
+  constructor(protected auth: WsiNgAuthService, protected wsi: WorldskillsAngularLibService, protected router: Route) { }
```

Comment:
Not so sure about the new name `WsiNgAuthService` here… I'm usually trying to avoid WSI in the public context like this. Maybe `WorldSkillsAuthService`?

Reply:
We have 2 auth services. the `WsiNgAuthService` is specific to the angular auth model. the `WsiAuthService` is more focused on our auth API. perhaps I can rename them as `NgAuthService` and `WorldSkillsAuthService` (or just `AuthService`

Reply:
Ah I see… Yeah I think `NgAuthService` and `WorldSkillsAuthService` would make sense then to make it clear.